src/Model/User.php

```diff
@@ -96,6 +96,59 @@ class User
 
 	private static $owner;
 
+	/**
+	 * Sends a welcome e-mail to the specified user
+	 *
+	 * @param array $user The user
+	 * @param string $password The user's password
+	 * @param string $preamble The user's password
```

Comment:
```suggestion
	 * @param string $password The user's password
	 * @param string $preamble
```

Maybe we can remove both comments?